app/src/main/java/com/raghav/spacedawn/ui/fragments/SearchArticleFragment.kt

```diff
@@ -40,7 +32,7 @@ class SearchArticleFragment : Fragment(R.layout.fragment_search_article) {
         binding = FragmentSearchArticleBinding.bind(view)
         setupRecyclerView()
 
-        searchArticlesAdapter.setOnItemClickListener {
+        articlesAdapter.setOnItemClickListener {
             val bundle = Bundle().apply {
                 putSerializable("article", it)
```

Comment:
It is advised to use the string literal here from a const in utils

Reply:
done